lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -504,7 +507,7 @@ def base_cmd
       [
         '@qemu_bin@ -enable-kvm -machine @machine_options@ ',
         '-m @memory@,maxmem=@max_memory@ -smp @cpu_count@,cores=@cpu_count@ ',
-        '-cpu @cpu_options@ -boot order=cd,menu=on ',
+        '-cpu @cpu_options@ -boot menu=on,splash-time=10000 ',
```

Comment:
Please add a comment to describe the situation where `splash-time` is needed.

Reply:
done